test/suite/hook_tiered.py

```diff
@@ -393,6 +393,16 @@ def setup_hooks(self):
 
 # Override some platform APIs for this hook.
 class TieredPlatformAPI(wthooks.WiredTigerHookPlatformAPI):
+    def __init__(self, arg=0):
```

Comment:
`arg=None` is probably clearer.  When `0` is used, it signals that this is probably an integer argument, and it's not.

Reply:
Fixed